protocol/communities/community_event_message.go

```diff
@@ -0,0 +1,257 @@
+package communities
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"sort"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/status-im/status-go/protocol/protobuf"
+)
+
+type CommunityAdminEvent struct {
+	AdminEventClock        uint64                                      `json:"clock"`
+	Type                   protobuf.CommunityAdminEvent_EventType      `json:"type"`
+	CommunityConfig        *protobuf.CommunityConfig                   `json:"community_config,omitempty"`
+	TokenPermission        *protobuf.CommunityTokenPermission          `json:"token_permissions,omitempty"`
+	CategoryData           *protobuf.CategoryData                      `json:"category_data,omitempty"`
+	ChannelData            *protobuf.ChannelData                       `json:"channel_data,omitempty"`
+	MemberToAction         string                                      `json:"member_to_action,omitempty"`
```

Comment:
We use camelCase for json names usually.

Reply:
Also, what is `MemberToAction`? The member to whom the action applies? If so, that name is a bit confusing.

Reply:
yes, for ban/unban, kick member